src/elasticsearch-service.ts

```diff
@@ -143,12 +140,22 @@ export const createIndex = async (
   try {
     const { body } = await client.indices.create({
       index,
-        body: {
-          mappings: mapping,
-          // eslint-disable-next-line @typescript-eslint/camelcase
-          settings: { number_of_shards: config.NUMBER_OF_SHARDS },
+      body: {
+        mappings: mapping,
+        settings: {
+          number_of_shards: config.NUMBER_OF_SHARDS,
+          analysis: {
+            analyzer: {
+              standard_nohtml_tags: {
+                type: 'custom',
+                tokenizer: 'standard',
+                char_filter: ['html_strip'],
+                filter: ['lowercase'],
+              },
+            },
+          },
```

Comment:
this should be part of the index configuration (alongside the mapping conf in the now called mapping-file), but it would require a refactoring of the deal service so it will come as a separate PR

Reply:
Where would the tests live for this?

Reply:
tests would live in the `src/__tests__/orchestrator.integration.spec.ts` but they are currently broken because they haven't been ported to the new, heavily AWS dependent, implementation.